code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -109,20 +109,22 @@
 //Step two - washing..... it's actually in washing machine code.
 
 //Step three - drying
-/obj/item/stack/sheet/wetleather/fire_act(exposed_temperature, exposed_volume)
-	..()
-	if(exposed_temperature >= drying_threshold_temperature)
-		wetness--
-		if(wetness == 0)
-			//Try locating an exisitng stack on the tile and add to there if possible
-			for(var/obj/item/stack/sheet/leather/HS in src.loc)
-				if(HS.amount < 50)
-					HS.amount++
-					src.use(1)
-					wetness = initial(wetness)
-					break
-			//If it gets to here it means it did not find a suitable stack on the tile.
-			var/obj/item/stack/sheet/leather/HS = new(src.loc)
-			HS.amount = 1
-			wetness = initial(wetness)
+/obj/item/stack/sheet/wetleather/fire_act(burn_level)
+	. = ..()
+	if(!wetness)
+		return
+	wetness--
+	if(wetness < 0)
+		return
```

Comment:
This whole block feels weird, but otoh is wet leather even obtainable?

Reply:
No idea, I just edited the proc to make it not entirely cursed